src/server/req_jobobit.c

```diff
@@ -633,14 +633,14 @@ rel_resc(job *pjob)
 	/* is there a rerun request waiting for acknowledgement that        */
 	/* resources (including licenses) are indeed released? Then ack it. */
 	if (pjob->ji_rerun_preq != NULL) { /* set only in req_rerun() */
+		if (pjob->ji_rerun_preq->rq_conn != PBS_LOCAL_CONNECTION)
+			conn = get_conn(pjob->ji_rerun_preq->rq_conn);
+
 		reply_ack(pjob->ji_rerun_preq);
 
 		/* clear no-timeout flag on connection */
-		if (pjob->ji_rerun_preq->rq_conn != PBS_LOCAL_CONNECTION) {
-			conn = get_conn(pjob->ji_rerun_preq->rq_conn);
-			if (conn)
-				conn->cn_authen &= ~PBS_NET_CONN_NOTIMEOUT;
-		}
+		if (conn)
```

Comment:
I know it's not your comment, but could you please expand this and explain why this flag needs to be cleared? Might save people some time. But it's up to you, I won't hold your check-in for this. Thanks